src/skmultiflow/meta/streaming_random_patches.py

```diff
@@ -0,0 +1,667 @@
+from copy import deepcopy
+from typing import List, Optional
+from _collections import deque
+
+import numpy as np
+
+from sklearn.preprocessing import normalize
+
+from skmultiflow.core import BaseSKMObject, ClassifierMixin, MetaEstimatorMixin, clone
+from skmultiflow.drift_detection.base_drift_detector import BaseDriftDetector
+from skmultiflow.trees import HoeffdingTreeClassifier
+from skmultiflow.drift_detection import ADWIN
+from skmultiflow.utils import check_random_state, get_dimensions
+from skmultiflow.metrics import ClassificationPerformanceEvaluator
+
+
+class StreamingRandomPatchesClassifier(BaseSKMObject, ClassifierMixin, MetaEstimatorMixin):
+    """ Streaming Random Patches ensemble classifier.
+
+    Parameters
+    ----------
+    base_estimator: BaseSKMObject or sklearn.BaseObject, \
```

Comment:
The `\` is required so `sphinx` recognizes the next line as part of the definition of the parameter. Otherwise, it takes it as part of the description breaking the format of the documentation.

Reply:
Thank you for clarifying! Every new day I discover I know less than I though about programming XD